Sources/Support/DebugUI/DebugViewSheetPresentation.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  DebugViewSheetPresentation.swift
+//
+//  Created by Nacho Soto on 5/30/23.
+
+#if DEBUG && canImport(SwiftUI) && !os(macOS)
+
+import SwiftUI
+
+@available(iOS 16.0, tvOS 16.0, *)
+extension View {
+
+    @ViewBuilder
+    func bottomSheet(
```

Comment:
I took this from my own app. It's necessary because `SwiftUI`'s sheets still don't expose everything that UIKit has. Maybe in iOS 17?